charts/civil-ccd/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 description: A Helm chart for civil-ccd App
 name: civil-ccd
 home: https://github.com/hmcts/civil-ccd-definition
-version: 0.0.21
+version: 0.0.22
```

Comment:
should we not also bump the civil-service dependency version here, to use 0.0.37?
That can also iinject the new env variable `WA_TOGGLE: WA3.5`